packages/docs/src/routes/docs/getting-started/index.mdx

```diff
@@ -21,33 +21,44 @@ Qwik is a new kind of framework that is [resumable](../concepts/resumable/index.
 
 ## Creating an app using the CLI
 
-The first step is to create an application. Qwik comes with a CLI that allows you to create a basic working skeleton of an application. We will use the CLI to create a blank starter so that you can quickly familiarize yourself with it.
+The first step is to create an application. Qwik comes with a CLI that allows you to create a basic working skeleton of an application. We will use the CLI to create a blank starter so that you can quickly familiarize yourself with it. Qwik supports NPM, yarn and pnpm.
 
 ### Run the Qwik CLI in your shell
+Choose the package manager you prefer and run the following command:
 
 ```shell
 npm create qwik@latest
+pnpm create qwik@latest
+yarn create qwik
 ```
 
-The CLI will guide you through an interactive menu to set the project-name and select one of the starters:
+The CLI will guide you through an interactive menu to set the project-name, select one of the starters and asks if you want to install the dependencies.
 
 After your new app is created, you will see an output like the following in your terminal:
 
 ```shell
-💫 Let's create a Qwik app 💫
+🐰 Let's create a Qwik app 🐇   v0.16.1
```

Comment:
this version will have to be constatly modified right?

Reply:
yep. thought about that one as well. i left it because the whole log output could change in a newer release and therefore it's eventually clearer when that state was generated. please let me know what's preferred 🙏

Reply:
Yeah that makes sense

So let's leave it and if it'll cause issues we'll remove it